Gemfile.lock

```diff
@@ -60,6 +60,7 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     autoprefixer-rails (9.8.6.1)
       execjs
+    bcrypt (3.1.15)
```

Comment:
@georgkloeck - this Gemfile.lock file has a merge conflict because it was changed on Master as well as your local branch.  That's why it's important for you to pull Master into your local feature branch before you push.  make sense?